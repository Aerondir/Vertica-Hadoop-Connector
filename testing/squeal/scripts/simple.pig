fs -copyFromLocal $script/foo.log simple/foo.log;
cat simple/foo.log;
x = LOAD 'simple/foo.log' USING PigStorage('|') as (a:int,b:int,c:int,d:chararray);
y = FILTER x BY a > 5;
z = FOREACH y GENERATE b*c as f;
z2 = DISTINCT z;
z3 = FOREACH z2 GENERATE f, f*f as g;
z4 = ORDER z3 by g;
z5 = LIMIT z4 3;
STORE z5 INTO 'simple/zzz';
cat simple/zzz;
fs -rmr simple;
