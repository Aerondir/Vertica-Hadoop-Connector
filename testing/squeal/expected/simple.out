10/10/18 23:56:04 INFO squeal.Squeal: Logging error messages to: /home/rvenkatesh/dev/Linux64/Test/squeal_smoke/pigshell.out
Constructing server
Configuring vertica connection
1|1|100|q
2|1|100|w
3|1|100|e
4|2|100|r
5|2|100|t
6|2|100|y
7|3|100|u
8|-7|100|i
9|3|100|o
10|4|100|p
Translating...
Running script schema refresh(SQL)
ROLLBACK: schema "squeal_PigLatin_simple_pig" does not exist
Time: 65ms
Running script sql setup(SQL)
Time: 57ms
Running script test sql(SQL)
Time: 60ms
Running script load3(PIG)
Use GenericOptionsParser for parsing the arguments. Applications should implement Tool for the same.
Total input paths to process : 1
Time: 18876ms
Running script populate vdata4(SQL)
Time: 248ms
Running script store vdata5(PIG)
Use GenericOptionsParser for parsing the arguments. Applications should implement Tool for the same.
Time: 18169ms
Total time: 37517ms
200	40000
300	90000
400	160000
Deleted hdfs://hermetic:9935/user/rvenkatesh/simple
